index.js

```diff
@@ -61,6 +61,29 @@ function main() {
       packageJson.workspaces = {
         packages: packagesConfig,
       };
+      const lockJsonKey = Object.keys(lockJson)
+      packageJson.resolutions = lockJsonKey.filter((dependency) => {
+        if(dependency.includes("@workspace:")){
+          return false;
+        }
+        if(dependency.includes(", ")){
+          return false;
+        }
+        if(!dependency.includes("@npm:")){
+          return false;
+        }
+        const [key] = dependency.split(":")
+        return lockJsonKey.every(dependency2=>{
+          if(dependency===dependency2){
+            return true;
+          }
+          return dependency2.split(",").map(dep=>dep.trim().split(":")[0]).every(dep=>dep !== key);
+        });
```

Comment:
could you add a comment explaining what this is doing?

Reply:
added
> we take only the dependencies that is not present multiple times in the lock file

it seems that is matching with resolutions, and not producing side effects
it is the result of empirical research, if someone have a deeper knowledge... :-)